### PR TITLE
DCM-111 Cohorts summary page

### PR DESCRIPTION
**Pull request by froberg**

### CODE REVIEW

**Comment by froberg** [comment]
ready for CR.
Have fun.

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Extensions/EnumExtensions.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@ public static class EnumExtensions
     {
         public static string GetDescription(this Enum enumerationValue)
         {
+            // ToDo: do we need?
```
Comment (leewadhams) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Extensions/EnumExtensions.cs]: Can be removed

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs [hunk 2]
```diff
@@ -51,22 +50,103 @@ public CommitmentOrchestrator(IMediator mediator, ICommitmentStatusCalculator st
...
-        public async Task<CommitmentListViewModel> GetAll(long providerId)
+        public async Task<CohortsViewModel> GetCohorts(long providerId)
         {
-            _logger.Info($"Getting all commitments for provider:{providerId}", providerId: providerId);
-
             var data = await _mediator.SendAsync(new GetCommitmentsQueryRequest
             {
                 ProviderId = providerId
             });
+            var commitmentStatus =
```
Comment (leewadhams) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs]: We should add some logging for the new methods

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs [hunk 3]
```diff
@@ -51,22 +50,103 @@ public CommitmentOrchestrator(IMediator mediator, ICommitmentStatusCalculator st
...
+                PageTitle = "Cohorts with employer",
+                PageId = "requests-with-employer",
+                PageHeading = "Cothorts with employer",
+                PageHeading2 = $"You have {data.Count} with employer for review:"
+            };
+        }
+
+        public async Task<CommitmentListViewModel> GetAllNewRequests(long providerId)
+        {
+            var d = await GetAllNew(providerId, RequestStatus.NewRequest);
```
Comment (leewadhams) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs]: I'm sure there's a more descriptive variable name than 'd'?

**Comment by leewadhams** [comment]
Just waiting on confirmation on the content for the flash messages.

**Pull request merged**